dev/local-environment/configurations/genesis/byron/genesis.json

```diff
@@ -62,6 +62,6 @@
         { "summand": "155381000000000" , "multiplier": "43946000000" }
     , "unlockStakeEpoch": "18446744073709551615"
     }
-, "protocolConsts": { "k": 10 , "protocolMagic": 42 }
```

Comment:
This was problematic. We had an inconsistency that is fatal in v10.5.1. k=5 is what we want as security parameter.